### DISCUSSION TITLE
Ist it possible to manage the network traffic allow list via the API?

### DISCUSSION THREAD

**Discussion by budachst** [CrowdStrike/falconpy#1001] [Q&A] [answered]
I was wondering, if the network allow list from the containment policy can be modified via the API? I searches the API docs and couldn't find any matching call for that. Maybe I missed something and someone else knows this?

Cheers!

**Comment by crowdstrikedcs** [comment, marked as answer]
Hi @budachst this is only available in the UI for now. There is no accessible API for altering allowed IPs for network containment. 

You may wish to leave an idea for our product team requesting this feature be added. These can be submitted from the Falcon UI Ideas section:
![image](https://github.com/CrowdStrike/falconpy/assets/43282232/0c93699e-0bcd-481e-ad97-c44f73468c18)

**Reply by budachst** [reply]
Hi @crowdstrikedcs,

thanks - good that I did look carefully enough, as I wasn't sure. I will make sure to file an idea for this.

**Comment by aaronherman** [comment]
IDEA-I-13182 🙏
